content/manuals/security/for-admins/single-sign-on/_index.md

```diff
@@ -10,29 +10,35 @@ aliases:
+When you enable SSO, Docker supports a Identity Provider Initiated SSO flow for user login. Instead of users authenticating using their Docker username and password, they are redirected to your identity provider's authentication page to sign in. Users must sign in to Docker Hub or Docker Desktop to initiate the SSO authentication process.
 
 The following diagram shows how SSO operates and is managed in Docker Hub and Docker Desktop. In addition, it provides information on how to authenticate between your IdP.
 
 ![SSO architecture](images/SSO.png)
 
 ## How to set it up
 
-1. Configure SSO by adding and verify your domain for your organization, then create an SSO connection with your IdP. Docker provides the Assertion Consumer Service (ACS) URL and Entity ID needed to establish a connection between your IdP server and Docker Hub.
-2. Test your connection by attempting to sign in to Docker Hub using your domain email address.
-3. Optionally, you can [enforce SSO](/security/for-admins/single-sign-on/connect/#optional-enforce-sso) sign-in.
-4. Complete SSO enablement. A first-time user can sign in to Docker Hub using their company's domain email address. They're then added to your company, assigned to an organization, and optionally assigned to a team.
+SSO is configured using the following steps:
+1. Configure SSO by creating and verifying a domain in Docker.
+2. Create your SSO connection in Docker and your IdP.
+3. Cross-connect Docker and your IdP.
+4. Test your connection.
+5. Provision users.
+6. Optional. Enforce sign-in.
+7. Manage your SSO configuration.
 
-## Prerequisites
+Once your SSO configuration is complete, a first-time user can sign in to Docker Hub or Docker Desktop using their company's domain email address. Once they sign in, they are added to your company, assigned to an organization, and if necessary, assigned to a team.
 
-* You must first notify your company about the new SSO login procedures.
-* Verify that your members have Docker Desktop version 4.4.2, or later, installed on their machines.
-* If your organization is planning to [enforce SSO](/security/for-admins/single-sign-on/connect/#optional-enforce-sso), members using the Docker CLI are required to [create a Personal Access Token (PAT)](/docker-hub/access-tokens/) to sign in instead of with a username and password. Docker plans to deprecate signing in to the CLI with a password in the future, so using a PAT will be required to prevent issues with authentication. For more details see the [security announcement](/security/security-announcements/#deprecation-of-password-logins-on-cli-when-sso-enforced).
-* Ensure all your Docker users have a valid user on your IdP with the same email address as their Unique Primary Identifier (UPN)
+## Prerequisites
+Before configuring SSO, ensure you meet the following prerequisites:
```

Comment:
There should be a blank line between the heading and the paragraph that follows it.

```plaintext
6.447 content/manuals/security/for-admins/single-sign-on/_index.md:36 MD022/blanks-around-headings Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "## Prerequisites"]
```

```suggestion
## Prerequisites

Before configuring SSO, ensure you meet the following prerequisites:
```